app/pages/project/instances/actions.tsx

```diff
@@ -111,12 +111,6 @@ export const useMakeInstanceActions = (
                   options.onDelete?.()
                   addToast({ title: `Deleting instance '${instance.name}'` })
                 },
-                onError: (error) =>
-                  addToast({
-                    variant: 'error',
-                    title: `Error deleting instance '${instance.name}'`,
-                    content: error.message,
-                  }),
```

Comment:
This was a bug on `main`. `confirmDelete` shows an error toast, so this one is redundant.